paws.common/R/credential_providers.R

```diff
@@ -322,7 +341,28 @@ get_container_credentials <- function() {
     return(NULL)
   }
 
-  return(metadata_response)
+  credentials_response_body <-
```

Comment:
`get_container_credentials()` should return... _credentials_. While this is technically changing the API, I think it's a meaningful improvement and shouldn't impact users, since these are internal functions

Reply:
makes sense to me :)